__tests__/data/pinterestConversion_input.json

```diff
@@ -81,6 +81,8 @@
                 "advertiserId": "429047995",
                 "appId": "429047995",
                 "enhancedMatch": true,
+                "enableDeduplication": true,
+                "deduplicationKey": "messageId",
```

Comment:
We don't need to update all the previous configs.

Reply:
ok